### ISSUE TITLE
A last removed service is not applied

### ISSUE THREAD

**Issue by uchiumih** [instatushq/out#41]
![remove-services](https://user-images.githubusercontent.com/51476259/121112031-d153d780-c84a-11eb-9362-fb805b42371a.gif)

This is an example.

1. Remove AWS, Slack and Docker.
2. Restart Instatus Out.
3. Docker is not removed.

**Comment by alisalahio** [comment]
Yeah you're correct! Same issue occurs with last **added** service.

I think it happens with the app store version. I'll try to fix this!
